Update plan-connect-design-concepts.md (#91030)
Azure/azure-content: articles/active-directory/hybrid/plan-connect-design-concepts.md

In express settings, the assumed choice for the attribute is userPrincipalName. If the userPrincipalName attribute does not contain the value you want your users to sign in to Azure, then you must choose **Custom Installation**.

>[!NOTE]

> >It is recommended as best practice that UPN prefix contains more than one character.

Reviewer: ```suggestion
>It's recommended as a best practice that the UPN prefix contains more than one character.
```
